apps/preview/nuxt/pages/showcases/Banners/VerticalBanner.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="flex flex-col gap-6 md:flex-row">
+    <div class="flex flex-col justify-between rounded-md md:items-center md:basis-1/2 bg-warning-200">
+      <div class="flex flex-col items-center p-4 text-center md:p-10">
```

Comment:
question: if we have `items-center` do we need `text-center` ?

Reply:
It is still needed to center text inside the paragraphs.